content/docs/operator-guide/create-a-stack.md

```diff
@@ -0,0 +1,327 @@
++++
+title="Create a stack"
+weight=2
++++
+
+Creating a custom [stack][stack] allows you to configure the base images for the build-time environment for your [builder][builder] and the run-time for your application.
+
+<!--more-->
+
+## Prerequisites
+
+Before we get started, make sure you've got the following installed: 
+
+{{< download-button href="https://store.docker.com/search?type=edition&offering=community" color="blue" >}} Install Docker {{</>}}
+
+
+## Creating a custom stack
+
+In this tutorial we will create a sample stack based on `Ubuntu Bionic`. To create a custom stack, we simply need to create customized build and run images. Let's see how we can do so!
+
+
+### Create a common base image
+
+Let's start by creating a base image containing layers that will be required by both the `build` and `run` images. In order to do this, switch to a clean workspace and create a `Dockerfile` as specified below:
+
+#### Defining the base
+We start with choosing `ubuntu:bionic` as our `base` image. Since we will be reusing these layers in both our build and run images we will be defining a common base image and leveraging [Docker's multi-stage builds](https://docs.docker.com/develop/develop-images/multistage-build/) to ensure this acts as the common base image for both our build-time and run-time environment.
+
+```Dockerfile
+# 1. Set a common base
+FROM ubuntu:bionic as base
+```
+
+#### Set required CNB information
+
+Next up we will be setting up the base image as required by the [Cloud-Native Buildpack specification][stack-spec] noted below.
+
+##### Specification
+
+**Labels**
+
+| Name                     | Description              | Format |
+| ------------------------ | ------------------------ | ------ |
+| `io.buildpacks.stack.id` | Identifier for the stack | String |
+
+**Environment Variables**
+
+| Name           | Description                            |
+| -------------- | -------------------------------------- |
+| `CNB_STACK_ID` | Identifier for the stack               |
+| `CNB_USER_ID`  | UID of the user specified in the image |
+| `CNB_GROUP_ID` | GID of the user specified in the image |
+<p class="spacer"></p>
+
+> **NOTE:** The **stack identifier** implies compatibility with other stacks of that same identifier. For instance, a custom stack may use
+> `io.buildpacks.stacks.bionic` as its identifier so long as it will work with buildpacks that declare compatibility with the
+> `io.buildpacks.stacks.bionic` stack.
+
+
+The `CNB_USER_ID` is the `UID`  of the user as which the `detect` and `build` steps are are run. The given user **MUST** be a non-root user
+and have it's home directly writeable. `CNB_GROUP_ID` is the primary `GID` of the above user.
+
+Let's update the `Dockerfile` to reflect the above specification.
+
+```Dockerfile
+# 1. Set a common base
+FROM ubuntu:bionic as base
+
+# ========== ADDED ===========
+# 2. Set required CNB information
+ARG cnb_uid=1000
+ARG cnb_gid=1000
```

Comment:
Are the `ARG`s necessary? If not, we could save a few lines by omitting them. IDK, it's fine to keep them though.

Reply:
Removed them.